cmd/cli/job.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2024 The Volcano Authors.
```

Comment:
It is recommended to remove the year #3668

Reply:
I think we can remove year in auto generated files: )